helm_deploy/values-cp-dev.yaml

```diff
@@ -10,6 +10,28 @@ generic-service:
     enabled: true
     host: prison-api-dev.prison.service.justice.gov.uk
     tlsSecretName: prison-api-cert
+    modsecurity_enabled: true
```

Comment:
unfortunately can't add comments to the snippet below as causes
```
Error: failed to parse values-cp-dev.yaml: error converting YAML to JSON: yaml: line 17: did not find expected key
```
so have to add above

Reply:
Might be an issue with using `toYaml ` in this line https://github.com/ministryofjustice/hmpps-helm-charts/blob/main/charts/generic-service/templates/ingress_cp.yaml#L24

The contents of "modsecurity_snippet" really is not yaml, it's a multiline string.